aea/agent_loop.py

```diff
@@ -263,10 +263,6 @@ async def _task_process_inbox(self) -> None:
         self.logger.info("[{}]: Start processing messages...".format(self._agent.name))
         while self.is_running:
             await inbox.async_wait()
-
-            if not self.is_running:  # make it close faster
-                return
-
```

Comment:
No longer needed?

Reply:
it's was additional i check, just decided to remove